.github/workflows/crd-sync-check.yaml

```diff
@@ -0,0 +1,34 @@
+# a simple boolean to check of a CRD sync check is needed
+name: CRD sync check
+
+on:
+  schedule:
+    - cron: '0 */2 * * *' # every 2 hours
+  workflow_dispatch:
+
+jobs:
+
+  crd-sync-check:
+    if: github.repository_owner == 'openstack-k8s-operators'
+    name: Check if 
```

Comment:
This is where the trailing whitespace checker is failing. Just need to remove the space after the if

Reply:
should be fixed now. thanks